src/reducers/message.js

```diff
@@ -0,0 +1,18 @@
+import { NOTE_ADD, NOTES_LOAD } from './note';
+
+const MESSAGE_SHOW = 'MESSAGE_SHOW';
+
+export const showMessage = msg => ({ type: MESSAGE_SHOW, payload: msg });
+
+export default (state = '', { type, payload }) => {
+  switch (type) {
+    case MESSAGE_SHOW:
+      return payload;
+    case NOTE_ADD:
```

Comment:
setting message to '' after the actions are done that are called in note.js. Otherwise the messages would never go away